config/fedora.yml

```diff
@@ -0,0 +1,4 @@
+user: fedoraAdmin
```

Comment:
Oops, these reviews seem to be for a PR that I closed first thing this morning. I will look at the comments though. Seems silly that Github would allow commenting on a closed PR. The current PR, which includes solr config is here: https://github.com/ualbertalib/pushmi_pullyu/pull/49

I'm indifferent as to whether there is one yaml or multiple (as long as the yaml tree doesn't get too branchy).

I thought we weren't going to use ActiveFedora (not sure there is any reason to), so the need to use separate YAML files may be moot.

Anyways, I need to read in some Fedora/Solr configuration somehow, so I'll continue down this road so that I'm not blocked by Shane's work on the config stuff (then I can switch over to his framework after he's done).